examples/natmod/btree/Makefile

```diff
@@ -32,6 +32,8 @@ SRC += $(addprefix $(realpath $(BTREE_DIR))/,\
 	mpool/mpool.c \
 	)
 
+CLEAN_EXTRA = "$(MOD).mpy"
```

Comment:
This is not really related to this PR.  Please put it in a separate commit, at least.

And maybe it can go in `py/dynruntime.mk` so it doesn't need to be repeated for each example?

Reply:
Good point!  The change in question is now limited to `py/dynruntime.mk` and was moved to its own commit.